api/CSSCounterStyleRule.json

```diff
@@ -287,13 +287,13 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "91"
```

Comment:
The following `webview_android` change is missing here. I can make that edit though.